benchmark/src/main/java/com/amazon/opendistroforelasticsearch/sql/benchmark/utils/load/mysql/MysqlDataTransformer.java

```diff
@@ -32,6 +43,63 @@
    */
   @Override
   public DataFormat transformData(String dataPath) throws Exception {
-    return null;
+    File path = new File(dataPath);
+    if (!path.exists() || !path.isDirectory()) {
+      throw new FileNotFoundException("Invalid Directory");
+    }
+
+    MysqlDataFormat result = new MysqlDataFormat();
+
+    // Create directory to store transformed json files
+    CommandExecution.executeCommand("mkdir " + dataPath + "mysql/");
+
+    // Create new json file for every  table / .tbl file
+    transformedDataPath = dataPath + "mysql/";
+
+    for (String tableName : MysqlTpchSchema.schemaMap.keySet()) {
+      File table = new File(dataPath + tableName + ".tbl");
+      if (!table.exists() || !table.isFile()) {
+        throw new FileNotFoundException(tableName + ".tbl not found");
+      }
+
+      FileReader fileReader = new FileReader(table);
+      BufferedReader bufferedReader = new BufferedReader(fileReader);
+
+      int tableDataFilesIndex = 1;
+      String filename = tableName + "_data_" + tableDataFilesIndex++ + ".csv";
+
+      BufferedWriter writer = new BufferedWriter(
+          new FileWriter(transformedDataPath + filename, true));
+
+      long tableLineIndex = 1;
```

Comment:
What is this code block doing - a comment would be helpful.

Reply:
added some comments